README.md

```diff
@@ -155,6 +357,33 @@ successfully deployed cosmsodb database and collections
 
 ```
 
+## Deploy
```

Comment:
since you used `deploy` also for the Azure resources is worth to specify we're talking about `Deploy new releases`